src/app/containers/BulletedList/index.test.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
+import BulletedListContainer from './index';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+describe('BulletedListContainer', () => {
+  describe('with data', () => {
+    const ltrData = {
```

Comment:
Can the fixtures be consolidated somehow?  We have the data here, similar looking data in `BulletListItem/index.test.js`, `convertToOptimoBlocks/blocks/list/fixtures.js`, as well as the standard CPS fixtures in the `data` directory

Reply:
Consolidated!